client/landing/gutenboarding/onboarding-block/signup-form/index.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+// /**
+//  * Internal dependencies
+//  */
+import {
+	usePasswordlessSignUp,
+	UsePasswordlessSignUpStatus,
+	Provider,
+	Client,
+} from '@automattic/authentication';
+import { Button, TextControl, Modal } from '@wordpress/components';
+import FormLabel from 'components/forms/form-label';
```

Comment:
Rather than pulling in `<FormLabel>` from Calypso, `<TextControl>` comes with it's own `label` prop. It looks like this rendered:

<img width="278" alt="image" src="https://user-images.githubusercontent.com/1500769/72690063-51f23500-3b7d-11ea-9aa9-c3c60e957dfb.png">